packages/grid_client/scripts/orchestrators/kubernetes_worker.ts

```diff
@@ -26,7 +26,8 @@ async function deleteWorker(client, worker) {
 }
 
 async function main() {
-  const name = "testk8s";
+  const name = "k8s" + generateString(8);
```

Comment:
please add a comment for the user to change the name to an existing cluster name so he can add the worker to it